src/orquestra/sdk/_base/_retry.py

```diff
@@ -0,0 +1,28 @@
+################################################################################
+# © Copyright 2023 Zapata Computing Inc.
+################################################################################
+import time
+from typing import Optional, Tuple, Type
+
+
+def retry(
```

Comment:
Praise: this looks like a useful function. Good to have it as separate piece of code

Reply:
+1. I wanted to suggest something similar when I've seen you discussing retries. Nicely done!